api/core/core.proto

```diff
@@ -183,6 +202,62 @@ message GetInventoryResponse {
     repeated InventoryEntry entries = 1;
 }
 
+message PolicyValidation{
```

Comment:
Nit: can we format these to aligned? In `vscode` you can use the `Align` command to get them all to line up.

Reply:
done